src/org/labkey/test/components/domain/DomainFormPanel.java

```diff
@@ -288,6 +306,9 @@ public void click()
             }
         };
 
+        protected WebElement exportFieldsButton = Locator.tagWithClass("div", "domain-toolbar-export-btn")
+                .findWhenNeeded(this);
```

Comment:
The add field button finder uses `.findWhenNeeded(DomainFormPanel.this)` so that it has the given panel in scope when looking for the button. Is that needed here as well for the case when there are multiple domain form panels on the page?

Reply:
The addFieldButton finder is enclosed in an enclosed type so it can override click(); the DomainFormPanel.this notation allows the enclosed class member to up-reference the containing class.  The exportFieldsButton doesn't need to do that, it's a direct member of the current DomainFormPanel instance.

Reply:
Got it. Thanks for explaining.